### ISSUE TITLE
How do we cite PhotonLibOS?

### ISSUE THREAD

**Issue by ErvinXie** [alibaba/PhotonLibOS#642]
Hello, I really appreciated your work and I am using PhotonLibOS in my new work, I want to cite your work in my paper. I see your paper on github, do you have anything like bib entry for me to cite?

**Comment by beef9999** [comment]
Maybe we can publish the article in arXiv.org?  @lihuiba @Coldwings

**Comment by lihuiba** [comment]
> Maybe publish the article in arXiv.org? @lihuiba @Coldwings

The article is not about Photon itself; it's about context switching.

**Comment by ErvinXie** [comment]
Thank you, I will cite PhotonLibOS like that. I am working on an LLM service system. I just use Photon to asynchronously load cache data from disk to memory.

**Comment by lihuiba** [comment]
Glad  to see that Photon is used for LLM. By "cache data", do you mean kvcache?

And I want to mention that, we have also implemented with Photon a high-performance data connector that can be used for loading data files for training, or loading/saving checkpoint, or loading model files. See https://www.alibabacloud.com/help/en/oss/developer-reference/oss-connector-overview/?spm=a2c63.p38356.help-menu-31815.d_3_3_8.62c4d579yxtBLQ

**Comment by ErvinXie** [comment]
Yes, we utilized Photon to load kvcache from disk in an asynchronous way. Thanks for your recommendation of data connector. Our work is about LLM inference system. I don't whether this data connector can be also used by an inference system?

**Comment by lihuiba** [comment]
> whether this data connector can be also used by an inference system?

Yes, the feature for inference systems to load model files will be released soon.

**Comment by lihuiba** [comment]
After your work is published, please give us a hyper link, and we will use it as a showcase of Photon. Thanks!

**Comment by ErvinXie** [comment]
@lihuiba Our paper was rejected, but we plan to publish it on arxiv in a few weeks. We utilize PhotonLibOS to asynchronously read and write offloaded KVCache on SSD. By the way, we are going to opensource the related code soon (Maybe in April this year) in our project, [Ktransformers](https://github.com/kvcache-ai/ktransformers.git). PhotonLibOS is a great project, it achieves nearly full speed without much coding effort. We could also carry out deeper cooperation in the future.

**Comment by lihuiba** [comment]
Glad to hear it helps! Looking forward to your publication!

**Comment by beef9999** [comment]
@ErvinXie Your team is already well-known. Please feel free to ask any questions in github or dingtalk. We’d love to see Photon play an important role in the storage scope of AI infra.
